api/src/services/organizations_v1/create_organization_invitation.py

```diff
@@ -129,5 +181,8 @@ def create_organization_invitation(
 
     db_session.add(invitation)
 
+    # Send invitation email (non-blocking)
```

Comment:
Non-blocking as in async, or non-blocking as in - won't cause any issues even if it fails? I think you mean the latter, right?